main/ipc.js

```diff
@@ -16,6 +18,13 @@ function setupIpcMain () {
   ipcMain.handle('saturn:getWebUrl', saturnNode.getWebUrl)
   ipcMain.handle('saturn:getFilAddress', saturnNode.getFilAddress)
   ipcMain.handle('saturn:setFilAddress', (_event, address) => saturnNode.setFilAddress(address))
+  // temp
+  ipcMain.handle('station:getFilAddress', saturnNode.getFilAddress)
+  ipcMain.handle('station:setFilAddress', (_event, address) => saturnNode.setFilAddress(address))
+  ipcMain.handle('station:getSawOnboarding', stationConfig.getSawOnboarding)
+  ipcMain.handle('station:setSawOnboarding', (_event) => stationConfig.setSawOnboarding())
```

Comment:
I find the name `SawOnboarding` a bit unusual. What alternatives we can consider? How about `OnboardingCompleted`?